health/health.d/hdfs.conf

```diff
@@ -56,7 +65,9 @@ component: HDFS
     every: 10s
      crit: $this > 0
     delay: down 15m multiplier 1.5 max 1h
-     info: number of datanodes which are currently dead
+     info: Number of datanodes that are currently dead. \
+           There are dead datanodes. \
+           Any data that was registered to a dead datanode is not available to HDFS anymore.
```

Comment:
We have `HDFS` in upper case here, but it is lowercase in the alarm with long line, I think we could use the same pattern for both alarms.

Reply:
"hdfs fsck"? that is a command